### ISSUE TITLE
I can here my self during call and also how can i manage speaker toggle on and off using this library?

### ISSUE THREAD

**Issue by krupalikevadiya** [blackuy/react-native-twilio-video-webrtc#510]
### Steps to reproduce
1.
2.
3.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Environment
- **Node.js version**:
- **React Native version**:
- **React Native platform + platform version**: iOS 9.0

### react-native-twilio-video-webrtc
**Version**: npm version or "master"

**Comment by slycoder** [comment]
To your first point, does this mean echo cancelation is not working?  It would help to know a little more about the device in question to dive deeper there.

There are some other issues with respect to toggling output device (ie #511). I'll follow up there.

**Comment by krupalikevadiya** [comment]
I'm canceling echo through   this.twillio_video.disableOpenSLES() and it is working but when i'm on call i can here my self after i stop to speak.
Detail of devices are : 
1.Pixel 3 with android 11
2.iphone xs

**Comment by slycoder** [comment]
Based on your description, it kind of sounds like the echo is coming back from the other party (i.e. if they mute do you still hear the echo)?  Is the second party also using echo cancellation? 

n.b. disableOpenSLES is only applicable to a few Android devices; I would avoid using it and instead relying on the default echo cancellation.

**Comment by krupalikevadiya** [comment]
Yes second party also use echo cancellation and i don't have issue with echo but main issue is i speak something on call i can hear exact what i speak after i stop means i can hear my self after i stop to speak.

**Comment by slycoder** [comment]
Yeah, I understand you there; I'm just trying to see if what you're hearing is your voice going through the second party, coming back through their mic and being played back to you (as opposed to a purely local loopback).

**Comment by krupalikevadiya** [comment]
yaa happens something like that what you say...can you please give me some solution for it?

**Comment by slycoder** [comment]
So as I said earlier, it sounds like their echo cancellation may not be working correctly.  Do you have more info on the settings they are using?  If they mute or use headphones, does that improve the issue?

**Comment by krupalikevadiya** [comment]
yes they use headphone and meeting is ongoing in team while testing..we are also tested by mute ourself in team but still can here ourself in call.

**Comment by krupalikevadiya** [comment]
@slycoder any updtae on it?

**Comment by yoramboccia-IR** [comment]
Any update on fixing echo cancellation?

**Comment by nicolasdevienne** [comment]
> I'm canceling echo through this.twillio_video.disableOpenSLES() and it is working but when i'm on call i can here my self after i stop to speak.
> Detail of devices are :
> 1.Pixel 3 with android 11
> 2.iphone xs

Same issue for me. When I'm on call I can here myself. Device : Pixel 2
